Client/Frontend/Browser/Browser.swift

```diff
@@ -164,15 +164,19 @@ class Browser: NSObject {
 
     var displayURL: NSURL? {
         if let url = webView?.URL ?? lastRequest?.URL {
-            if !AboutUtils.isAboutHomeURL(url) {
+            if !AboutUtils.isAboutURL(url) {
                 if ReaderModeUtils.isReaderModeURL(url) {
```

Comment:
These two conditions together don't make sense as written: in English, you're saying, "If this is not an about: URL and is an about:reader URL". It works because your `isAboutURL` implementation is a bit broken; I'd recommend fixing that method and then reordering these conditions appropriately.